octopus-server/src/main/java/octopus/teamcity/server/OctopusGenericRunType.java

```diff
@@ -33,25 +32,28 @@ public String getType() {
     return OctopusConstants.GENERIC_RUNNER_TYPE;
   }
 
+  @NotNull
```

Comment:
do we want to be doing "nonull" annotations? I was _trying_ to avoid them

Reply:
Removed following conversation. `@Nullable` usages also removed for consistency